core/src/main/java/com/linecorp/armeria/server/VirtualHostAnnotatedServiceBindingBuilder.java

```diff
@@ -67,271 +54,211 @@
             Iterable<? extends RequestConverterFunction> requestConverterFunctions) {
-        requireNonNull(requestConverterFunctions, "requestConverterFunctions");
-        requestConverterFunctionBuilder.addAll(requestConverterFunctions);
-        return this;
+        return (VirtualHostAnnotatedServiceBindingBuilder) super.requestConverters(requestConverterFunctions);
     }
 
     @Override
     public VirtualHostAnnotatedServiceBindingBuilder useBlockingTaskExecutor(boolean useBlockingTaskExecutor) {
-        this.useBlockingTaskExecutor = useBlockingTaskExecutor;
-        return this;
+        return (VirtualHostAnnotatedServiceBindingBuilder)
+                super.useBlockingTaskExecutor(useBlockingTaskExecutor);
     }
 
-    /**
-     * Sets the delimiter for a query parameter value. Multiple values delimited by the specified
-     * {@code delimiter} will be automatically split into a list of values.
-     *
-     * <p>It is disabled by default.
-     *
-     * <p>Note that this delimiter works only when the resolve target class type is collection and the number
-     * of values of the query parameter is one. For example with the query delimiter {@code ","}:
-     * <ul>
-     *     <li>{@code ?query=a,b,c} will be resolved to {@code "a"}, {@code "b"} and {@code "c"}</li>
-     *     <li>{@code ?query=a,b,c&query=d,e,f} will be resolved to {@code "a,b,c"} and {@code "d,e,f"}</li>
-     * </ul>
-     */
-    @UnstableApi
+    @Override
     public VirtualHostAnnotatedServiceBindingBuilder queryDelimiter(String delimiter) {
-        this.queryDelimiter = requireNonNull(delimiter, "delimiter");
-        return this;
+        return (VirtualHostAnnotatedServiceBindingBuilder) super.queryDelimiter(delimiter);
     }
 
     @Override
-    public VirtualHostAnnotatedServiceBindingBuilder requestTimeout(Duration requestTimeout) {
-        defaultServiceConfigSetters.requestTimeout(requestTimeout);
-        return this;
+    public VirtualHostAnnotatedServiceBindingBuilder decorator(
+            DecoratingHttpServiceFunction decoratingHttpServiceFunction) {
+        return (VirtualHostAnnotatedServiceBindingBuilder) super.decorator(decoratingHttpServiceFunction);
     }
 
     @Override
-    public VirtualHostAnnotatedServiceBindingBuilder requestTimeoutMillis(long requestTimeoutMillis) {
-        defaultServiceConfigSetters.requestTimeoutMillis(requestTimeoutMillis);
-        return this;
+    public VirtualHostAnnotatedServiceBindingBuilder decorator(
+            Function<? super HttpService, ? extends HttpService> decorator) {
+        return (VirtualHostAnnotatedServiceBindingBuilder) super.decorator(decorator);
     }
 
+    @SafeVarargs
     @Override
-    public VirtualHostAnnotatedServiceBindingBuilder maxRequestLength(long maxRequestLength) {
-        defaultServiceConfigSetters.maxRequestLength(maxRequestLength);
-        return this;
+    public final VirtualHostAnnotatedServiceBindingBuilder decorators(
```

Comment:
nit
```suggestion
    public VirtualHostAnnotatedServiceBindingBuilder decorators(
```

Reply:
This is actually necessary for `@SafeVarargs` 😄